modules/main-rules/pages/transportation.adoc

```diff
@@ -1,59 +1,80 @@
+These abilities indicate units that can carry -- transport -- other units across the battlefield.
+These abilities have some features in common, but also differ in some important ways.
 
-Both the {transport-x} and {hitch-x} abilities indicate units that can carry -- transport -- other units across the battlefield.
-But they work in slightly different ways.
+== Embark and disembark
+To describe the actions that relate to the units that you transport we use the terms *embark* and *disembark*.
+Where we refer to these actions, keep these additional points in mind:
 
-== Transportation by {transport-x} units
+Embark:: 
+You may embark a unit only when it is completely within 5 cm of the transportation unit.
+Disembark::
+When you disembark a unit, place it completely within 5 cm of the transportation unit.
+
+[NOTE.e40k]
+.Embark and disembark
+====
+The transportation rules in _Epic 40,000_ leave a lot of room for interpretation, and they aren't entirely clear on some of the commonalities and differences between _Transport_ and _Hitch_.
+Nor do they account well for disparities that stem from the range of different base sizes that Epic enthusiasts use for their units these days.
 
-Units that have the {transport-x} ability can carry other units across the battlefield.
-They can only transport Infantry units -- except where the rules or army lists say otherwise.
+We've changed the conditions or embarkation and disembarkation and integrated them with the {transport-x} and {hitch-x} abilities to minimise these problems.
+====
+
+NOTE: To be clear, a transportation unit does not gain the abilities of any embarked units.
+// TODO: However, some abilities, or parts of abilities, can still operate from within a transportation unit. See comment further down, in relation to 'no other actions with embarked units'.
+
+== Transportation by {transport-x} units
 
-To embark or disembark a unit:
+To embark or disembark one or more units:
 
 * You must sacrifice 5 cm from the movement of the {transport-x} unit.
 * You must sacrifice 5 cm from the movement of the unit that you embark or disembark.
 * You may embark and disembark units in the same turn, the same phase or even in the same move -- so long as all of the involved units have enough movement.
 
-You may perform no other actions with units that are embarked -- so, you cannot shoot with them, for example.
+You may perform no other actions with embarked units -- so, you cannot shoot with them, for example.
 Obviously, once you disembark them then the normal rules apply again.
+// TODO: I think we might need to refine this because 'action' might be ambiguous -- for example, shouldn't the 'stubborn' part of the Supreme Commander ability still be allowed even while the unit is inside transportation, while the 're-roll Assault resolution die' part would not?
```

Comment:
I don't see a problem there, I can't imagine that anyone would see it any other way. That's not what I mean to highlight.

My point is more fundamental: "You may perform no other actions" might be ambiguous for some readers.

Arguably, in a broad sense some abilities imply 'action' of some kind. So, might some readers take that statement to mean that special abilities are wholly inoperable while a unit is embarked within a transport unit?

Whereas in fact we intend that appropriate parts of abilities remain operable where there is not an obvious contradiction. The Supreme Commander example I gave highlights an instance where part of the ability would apply and part would not (because the SC isn't an active participant in the assault if he's inside the transport unit and not outside fighting).

Reply:
> My point is more fundamental: "You may perform no other actions" might be ambiguous for some readers.
> 
> Arguably, in a broad sense some abilities imply 'action' of some kind. So, might some readers take that statement to mean that special abilities are wholly inoperable while a unit is embarked within a transport unit?

Have you got any examples? The only special abilities I can think of at the moment that involve an 'action' rather than a static ability is playing Psychic Blast Fate Cards from a Psyker, or Save stopping a hit.

Reply:
Well, the one that sprung to mind was the re-roll part of Supreme Commander, which is why I gave it as an example. I didn't dig into it further, and I may have a misplaced nag from EA, but yes, part of the Psyker ability and perhaps the Medic ability might be construed as 'actions' — along with possible future abilities we haven't dreamed up yet.

I wonder if some readers — particularly if English is not their first language and they conflate concepts like 'action' and 'effect' — *might* construe that statement to cancel parts of an ability that should remain operable.

My note was more query than statement — I'm unsure and could be overthinking it. If, with the above in mind, you don't agree that the statement could be ambiguous, then I'm happy to drop it.

Reply:
I think (hope) that this could be overthinking it!

Reply:
Righto, I'll delete that note. It's not like we can't change it if it ever does prove to be problematic for some people.
```suggestion
```